src/reducers/match.js

```diff
@@ -51,7 +51,12 @@ export default (state = initialState, action) => {
 export const getMatch = state => state.app.match;
 export const getMatchData = state => state.app.match.match;
 export const getMatchLoading = state => state.app.match.loading;
-export const getMatchPlayers = state => state.app.match.match.players;
+
+export const getMatchPlayers = (state, team = '') =>
```

Comment:
The boolean wont work since I need it for 3 things, all, radiant and dire.

Why do I move this function to Match component but not the others?